src/node_env_var.cc

```diff
@@ -316,6 +316,24 @@ Maybe<bool> KVStore::AssignFromObject(Local<Context> context,
   return Just(true);
 }
 
+// TODO(bnoordhuis) Not super efficient but called infrequently. Not worth
```

Comment:
For education purposes: Why is the following code `not super efficient`?

Reply:
Repeated locking of `per_process::env_var_mutex` in `RealEnvStore::Get()`. Probably negligible to the point of immeasurable but I've been a C programmer for too long to not have that niggle at my back of mind.